addons/sale_timesheet/views/project_task_views.xml

```diff
@@ -33,7 +33,7 @@
                             <field name="company_id" invisible="1"/>
                             <field name="project_id" invisible="1"/>
                             <field name="employee_id" options="{'no_create': True}"/>
-                            <field name="timesheet_product_id" attrs="{'column_invisible': [('parent.sale_order_id', '!=', False)]}" invisible="1"/>
+                            <field name="timesheet_product_id" invisible="1"/>
```

Comment:
Is it still needed then?

I also fin an occurence in project.quick_create_task_form where it is also invisible. And it is not shown in the override in fsm if I am correct. Do wee need to keep them?

Reply:
Indeed, this field can be remove because is no longer used in tree view of mappings when the pricing type is 'employee rate'.
But if you said the timesheet_product_id field in project.project, it is override in fsm.
In project.task, I'm not sure but I think this field is not used and contains only the one in its project, I must check. 🤔